source/common/tls/aws_lc_compat.h

```diff
@@ -0,0 +1,3 @@
+#ifdef OPENSSL_IS_AWSLC
```

Comment:
Please add an explanatory comment at the top of the file describing what it does, why it needs to exist, some guidance on when things need to be added to the file, and maybe a link to the aws-lc github repo.